documentation/Installation.md

```diff
@@ -20,7 +20,7 @@ Saffire. Once the box has started, issue the following:
 
     vagrant ssh
     cd /vagrant
-    sh build.sh
+    sudo ./build.sh -i
```

Comment:
> I assume [sudo is required] for installing globally, but not sure if that works correctly yet.

Yep, that's exactly right, and it seemed to work fine during my testing.

The reason I changed this is the text below reads "This will compile and install Saffire", but without the `-i` switch, it _doesn't_ install.

I took the "Installation on your local system" section to be completely separate, so left those references to the build path alone. There are no full-path references to `/vagrant/build/release/bin/saffire` that I can see.